roles/keystone/tasks/main.yml

```diff
@@ -18,35 +18,70 @@
   file: dest=/etc/keystone state=directory
 
 - name: keystone log dir
-  file: dest=/var/log/keystone state=directory mode=0755 owner=keystone
-        group=keystone
-
-- name: install keystone service
-  upstart_service: name=keystone user=keystone cmd=/usr/local/bin/keystone-all
-                   config_dirs=/etc/keystone
+  file: dest=/var/log/keystone state=directory mode=0755 owner=www-data
+        group=www-data
```

Comment:
Maybe in the future, but as I understand it, mod_wsgi sites all run in the same process, which likely means they all run as the same user. For this POC I didn't want to spend time trying to sort that out.